impeller/entity/entity_pass.cc

```diff
@@ -409,9 +409,8 @@ bool EntityPass::Render(ContentContext& renderer,
     return true;
   });
 
-  ClipCoverageStack clip_coverage_stack = {ClipCoverageLayer{
-      .coverage = Rect::MakeSize(root_render_target.GetRenderTargetSize()),
-      .clip_depth = 0}};
+  EntityPassClipStack clip_stack = EntityPassClipStack(
+      Rect::MakeSize(root_render_target.GetRenderTargetSize()));
```

Comment:
This patch moves to a single list of tracked replay clips. So when replaying clips we need to be careful that only clips which impact the current pass are replayed, otherwise there may be perf issues if StC is turned on and fidelity bugs if StC is turned off.

Reply:
Oh, taking a closer look, maybe you're already handling this correctly. I see the state is held in a per-pass vector.